src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19313.cs

```diff
@@ -0,0 +1,30 @@
+#if TEST_FAILS_ON_IOS && TEST_FAILS_ON_CATALYST //https://github.com/dotnet/maui/issues/19313
+using System;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Tests.Issues;
+
+public class Issue19313 : _IssuesUITest
+{
+	public Issue19313(TestDevice device) : base(device)
+	{
+	}
+
+	public override string Issue => "Checkbox broken in Android";
+
+    [Test]
+	[Category(UITestCategories.CheckBox)]
+	public void CheckBoxWithTapGestureShouldBeChecked()
+	{
+		App.WaitForElement("MauiCheckBox");
+		App.Tap("MauiCheckBox");
+
+		var text1 = App.WaitForElement("MauiLabel").GetText();
+
+        Assert.That(text1,Is.EqualTo("Tapped"));
+        VerifyScreenshot();
```

Comment:
nit: Maybe we don't need a screenshot and instead you can do something similar to the other tests and add a label for checked changed?